.github/workflows/guestfs_index.yaml

```diff
@@ -7,6 +7,22 @@ on:
     types: [created]
 
 jobs:
+  always_run:
+    runs-on: ubuntu-latest
+    defaults:
+      shell: bash
+    steps:
+      - name: Always run
+        env:
+          GITHUB_EVENT: "${{ github.event }}"
+        # NOTE:BUG https://github.community/t/workflow-is-failing-if-no-job-can-be-ran-due-to-condition/16873/2
+        run: |
+          echo "This job is included because GitHub Actions marks the workflow as having "\
+               "'failed at startup' if no jobs are run, and the other job in this workflow "\
+               "is likely to be skipped"
+          python -c "import os;print(os.environ['GITHUB_EVENT'])"
```

Comment:
This will most likely run with Python 2.7, and would be relying on an implicit `from __future__ import print_function`.

Reply:
```suggestion
          python3 -c "import os;print(os.environ['GITHUB_EVENT'])"
```